internal/testutil/testutil.go

```diff
@@ -86,6 +86,22 @@ func GetAvailableLocalIPv6Address(t testing.TB) string {
 	return endpoint
 }
 
+func GetAvailableLocalIPv6AddressPrometheus(t testing.TB) *config.Prometheus {
+	address := GetAvailableLocalIPv6Address(t)
+	host, port, err := net.SplitHostPort(address)
+	if err != nil {
+		return nil
+	}
+	portInt, err := strconv.Atoi(port)
+	if err != nil {
+		return nil
+	}
+	return &config.Prometheus{
+		Host: &host,
+		Port: &portInt,
+	}
+}
```

Comment:
Not sure this needs to be here, can be a private func in service_test

Reply:
sure I can move it

Reply:
done. moved `GetAvailableLocalAddressPrometheus` at the same time as its only used in `service` and it's an "internal" package